site/en/docs/extensions/mv3/manifest/background/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Manifest - background"
+date: 2023-05-24
+updated: 
+description: Reference documentation for the background property of manifest.json.
+---
+
+An optional manifest key used to specify a javascript file as the extension service worker. A service worker is a persistent script that acts as the extension's main event handler. For more information, visit the [more comprehensive introduction to service workers](docs/extensions/mv3/service_workers/#manifest).
+
+```json
+{
+  // ...
+   "background": {
+      "service_worker": "service-worker.js"
```

Comment:
```suggestion
      "service_worker": "service-worker.js",
      "type": "module"
```

Reply:
This is a rare case where duplication is acceptable. The content at the end of the link optimizes for readers who need to understand extension service workers in whole or in part. A page like this is optimized for quick lookups. Ideally, there's more content that should be on a reference page, but this will do for now.

Reply:
I believe we should state explicitly when a field is optional, such as "type":"module".

Reply:
I agree.